SofaKernel/modules/SofaSimpleFem/HexahedronFEMForceField.inl

```diff
@@ -1271,6 +1271,29 @@ void HexahedronFEMForceField<DataTypes>::addKToMatrix(const core::MechanicalPara
 
 
 
+template<class DataTypes>
+void HexahedronFEMForceField<DataTypes>::computeBBox(const core::ExecParams* params, bool onlyVisible)
+{
+    if( !onlyVisible ) return;
+
+    helper::ReadAccessor<DataVecCoord> x = this->mstate->read(core::VecCoordId::position());
+
+    static const Real max_real = std::numeric_limits<Real>::max();
+    static const Real min_real = std::numeric_limits<Real>::min();
```

Comment:
warning `std::numeric_limits::min()` is not what you want, you want either `-std::numeric_limits::max()` or in c++11 `std::numeric_limits::lowest()`